business-registry-dashboard/app/locales/fr-CA.ts

```diff
@@ -151,6 +151,18 @@ export default {
     businessAdd: {
       title: "Error lors de l'ajout de l'Entreprise Existante",
       description: "Une erreur s'est produite lors de l'ajout de votre entreprise. Veuillez réessayer."
+    },
+    magicLinkUnauthorized: {
```

Comment:
I used google translate for these. I don't speak French haha. Sev, please if you can have a quick look and confirm that translations are precise, that would awesome!

Reply:
Looks OK to me.

Should the titles be title-cased?

What's the actual French name of Business Registry (line 165)?

Reply:
Ah, see line 360.

Reply:
What do you mean by title-cased Sev here?

Reply:
"You Could Google It and Find Some Official Definitions, but This Sentence Is an Example of Title Case."

Reply:
Ahhh, I thought it's something specific to French 😆 Yes I think you are right. Will fix that.

Reply:
Sev, I title cased these. This will eventually be passed by the UX people so if they don't like it, they'll point it out but from what I saw already here in this file, everything is title-cased as well.

Reply:
Thanks. I have nothing further to add for this conversation (but I cannot resolve it).